auto_rx/autorx/aprs.py

```diff
@@ -89,8 +89,8 @@ def telemetry_to_aprs_position(
 
     # TODO: RS41 Burst Timer
 
-    # Add on auto_rx version
-    _aprs_comment += " auto_rx v" + auto_rx_version
+    # Add on auto_rx at the end
+    _aprs_comment += " auto_rx"
```

Comment:
I believe this will still be different depending on software variant/version. The comment should only contain things from the sonde, not the gateway software, so that each gateway would send the exact same comment string.

Reply:
I'll fix this up, but note that autorx will still be sending to a different destination 
`      APRARX  VK5QI's radiosonde tracking` so I'm not sure if this will be that helpful for deduplication ?

Reply:
That will need to be fixed as well. We'll have to assign a common destination callsign for sondes at https://github.com/aprsorg/aprs-deviceid and use it instead of a gateway-specific destination call.